src/lib/ecore/ecore_time.c

```diff
@@ -1,22 +1,15 @@
 #ifdef HAVE_CONFIG_H
 # include <config.h>
 #endif
-
 #define EFL_LOOP_PROTECTED
-
 #include <stdlib.h>
-#ifndef _WIN32
-# include <sys/time.h>
-#else
+#ifndef _MSC_VER
 # include <evil_private.h>
 #endif
-
 #if defined(__APPLE__) && defined(__MACH__)
 # include <mach/mach_time.h>
 #endif
-
```

Comment:
I think those 1 blank lines here make for a good code separation, making it easier to read.